docs/docs-beta/docs/getting-started/concepts.md

```diff
@@ -0,0 +1,641 @@
+---
+title: "Concepts"
+sidebar_position: 200
+---
+
+```mermaid
+%%{
+  init: {
+    'theme': 'base',
+    'themeVariables': {
+      'primaryColor': '#4F43DD',
+      'primaryTextColor': '#fff',
+      'primaryBorderColor': '#231F1B',
+      'lineColor': '#DEDDFF',
+      'secondaryColor': '#BDBAB7',
+      'tertiaryColor': '#BDBAB7'
+    }
+  }
+
+}%%
+  graph TD
+
+    AssetCheck(AssetCheck)
+    %% click AssetCheck href "concepts#asset-check"
```

Comment:
Currently disabling `click` since it ruins the padding:

![Screenshot 2025-02-07 at 11 04 05 AM](https://github.com/user-attachments/assets/59f45ae8-4eed-4952-83f6-806e70c4af39)